.gitignore

```diff
@@ -9,10 +9,22 @@ reports/
 pynest/pynestkernel.cxx
 *.la
 *.lo
+*.so
 *.log
 *.o
 *.pyc
-*.so
+
+Makefile
+!/doc/Makefile
```

Comment:
This file was [removed](https://github.com/nest/nest-simulator/pull/1843/files#diff-f48ddfd1eec38c4d39f84e195259371f8397cd30146f46b113bdcb4590ed262c) in #1843
```suggestion
```